R/predictors.R

```diff
@@ -59,7 +59,9 @@
 #' @keywords internal
 getPred_baseflow <- function(data, metadata, method=c("hysep","1p digital filter","2p digital filter"), 
                              da, select, alpha, BFImax, ...) {
-  
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  is.Date=is.POSIXt=hysep=NULL
```

Comment:
definitely not - see previous comment for is.Date and is.POSIXt. For hysep, i actually meant to change this last weekend but seem to have not done that. could you please comment out the entire hysep block in the switch statement, from 71-110? I'd rather make hysep unavailable than commit ourselves to a whole new package dependency just for this one option.